crates/turbopack-ecmascript/src/tree_shake/util.rs

```diff
@@ -492,11 +492,11 @@ pub fn should_skip_tree_shaking(m: &Program, special_exports: &[RcStr]) -> bool
 }
 
 #[derive(Default)]
-struct UseServerFinder {
+struct ShouldAbort {
```

Comment:
Could you rename that to `ShouldSkip`?

"Abort" as word could have a negative meaning in some contexts so we want to avoid it in general